package.json

```diff
@@ -42,21 +42,21 @@
     "test:coverage:ci": "cross-env NODE_ENV=test BABEL_ENV=coverage nyc --reporter=lcov mocha 'packages/**/*.test.{js,ts,tsx}' 'docs/**/*.test.{js,ts,tsx}'",
     "test:coverage:html": "cross-env NODE_ENV=test BABEL_ENV=coverage nyc --reporter=html mocha 'packages/**/*.test.{js,ts,tsx}' 'docs/**/*.test.{js,ts,tsx}'",
     "test:e2e": "cross-env NODE_ENV=production pnpm test:e2e:build && concurrently --success first --kill-others \"pnpm test:e2e:run\" \"pnpm test:e2e:server\"",
-    "test:e2e:build": "webpack --config test/e2e/webpack.config.js",
-    "test:e2e:dev": "concurrently \"pnpm test:e2e:build --watch\" \"pnpm test:e2e:server\"",
-    "test:e2e:run": "mocha --config test/e2e/.mocharc.js 'test/e2e/**/*.test.{js,ts,tsx}'",
+    "test:e2e:build": "pnpm release:build && vite build --config test/e2e/vite.config.js",
+    "test:e2e:dev": "vite --config test/e2e/vite.config.js -l info",
+    "test:e2e:run": "VITEST_ENV=chromium vitest --project e2e",
     "test:e2e:server": "serve test/e2e -p 5001",
     "test:karma": "cross-env NODE_ENV=test karma start test/karma.conf.js",
     "test:karma:profile": "cross-env NODE_ENV=test karma start test/karma.conf.profile.js",
     "test:regressions": "cross-env NODE_ENV=production pnpm test:regressions:build && concurrently --success first --kill-others \"pnpm test:regressions:run\" \"pnpm test:regressions:server\"",
-    "test:regressions:build": "pnpm release:build && webpack --config test/regressions/webpack.config.js",
-    "test:regressions:dev": "concurrently \"pnpm test:regressions:build --watch\" \"pnpm test:regressions:server\"",
-    "test:regressions:run": "mocha --config test/regressions/.mocharc.js --delay 'test/regressions/**/*.test.js'",
+    "test:regressions:build": "pnpm release:build && vite build --config test/regressions/vite.config.js",
+    "test:regressions:dev": "vite --config test/regressions/vite.config.js -l info",
+    "test:regressions:run": "VITEST_ENV=chromium vitest --project regressions",
     "test:regressions:server": "serve test/regressions -p 5001",
     "test:unit": "cross-env NODE_ENV=test mocha 'packages/**/*.test.{js,ts,tsx}' 'docs/**/*.test.{js,ts,tsx}'",
-    "test:jsdom": "cross-env NODE_ENV=test VITEST_ENV=jsdom vitest",
-    "test:chromium": "cross-env NODE_ENV=test VITEST_ENV=chromium vitest",
-    "test:firefox": "cross-env NODE_ENV=test VITEST_ENV=firefox vitest",
+    "test:jsdom": "cross-env NODE_ENV=test VITEST_ENV=jsdom vitest --project @base-ui-components/react --project docs",
```

Comment:
These all need to be filtered with `--project` now that `test` also includes pnpm workspaces

I noticed that tests in `docs` weren't being run at all before explicitly specifying the project flag